about.html

```diff
@@ -35,7 +35,7 @@
   <body>
     <header>
       <nav class="container">
-        <a href="index.html" class="my-logo d-none d-md-flex">
+        <a href="index.html" class="my-logo">
           <figure>
             <img
```

Comment:
@topeogunleye Please kindly take your time to look at this correction you requested again, I'd suggest you clone this repo on your machine, make the changes and see for yourself.
There's an active state on the "About", that indicated that the user is on the About page, if I changed that to this: `<a href="index.html">Home</a> ` as you suggested, it become more confusing, the logo has already been linked to the home page to enable the user to navigate back to the homepage.

I no reason why i have to changed the About to Home when the user is on the About page.

Reply:
@Errytagedesign, when the user is on about page, the home page link should be an about page link instead because why is there an about page link on the about page. it simply goes back to the about page

Reply:
@Errytagedesign 

Hello @student, thank you for your reply. I understand your point of view, but I still think that you should change the About link to Home on the About page. Here are some reasons why:

- The project requirements clearly state that you should have a Home link on the About page, and vice versa. This is a common practice in web design, and it helps the user to know where they are and where they can go.
- The active state on the About link is not enough to indicate that the user is on the About page, because it is not very noticeable and it can be confused with a hover state. A Home link would be more clear and intuitive for the user.
- The logo link to the home page is not obvious for the user, because it is not a standard convention and it does not have any text or icon that indicates its functionality. A Home link would be more accessible and user-friendly.

I hope you can see my perspective and reconsider your decision. I suggest you look at some examples of websites that have a Home link on their About page, such as https://www.apple.com/about/ or https://www.microsoft.com/en-us/about. You can also check some resources on web design best practices, such as https://www.nngroup.com/articles/homepage-links/ or https